src/Components/Form/Form.js

```diff
@@ -1,154 +1,243 @@
-import React, { Component } from 'react';
-import { getAllQuestions } from '../../apiCalls'
-import './Form.scss';
+import React, { Component } from "react";
+import { getAllQuestions } from "../../apiCalls";
+import "./Form.scss";
 import { RiHomeSmileLine } from "react-icons/ri";
-import { Link } from 'react-router-dom';
+import { Link, Redirect } from "react-router-dom";
 import { CgUserlane } from "react-icons/cg";
 export class Form extends Component {
   constructor(props) {
-    super(props)
+    super(props);
     this.state = {
+      history:[],
       prompts: [],
       allAnswers: [],
       currentAnswers: [],
       questionsPerActivity: [],
-    }
+    };
   }
 
   componentDidMount = async () => {
-    this.setState({prompts: await getAllQuestions()})
+    this.setState({ prompts: await getAllQuestions() });
+  };
+
+  setHistory = () =>{
+    this.setState({history:[...this.state.history, this.state]})
+   this.props.setHistory()
+  }
+      
+  goBack = (event) =>{
+    event.preventDefault()
+    this.setState(this.state.history[this.state.history.length - 1])
+    this.props.goBack()
   }
```

Comment:
Pretty much identical to the app history management functions

Reply:
I was wondering why this looked like everything was duplicated from App.  Thanks for clarifying!

Reply:
This is a great addition thanks!